### DISCUSSION TITLE
Closing lists when pressing enter twice

### DISCUSSION THREAD

**Discussion by evilmastermind** [facebook/lexical#6618] [Q&A]
Hi! I saw in the playground that, when pressing enter on an empty list item, the list is automatically "closed" and a paragraph is created instead. However, the rich-text editor I created on my own project doesn't do that (I'm using a non-react build at the moment).

Is this a feature already built in lexical that I could import/enable, or is it something I need to build myself?

Thank you!

**Comment by FerdinandObermeier** [comment]
In case you are still searching for a solution: This https://github.com/facebook/lexical/issues/4893#issuecomment-1680590217 does the trick.
